azure-pipelines-samples.yml

```diff
@@ -107,6 +257,79 @@ steps:
       createLogFile: true
     condition: succeededOrFailed()
 
+  - task: CopyFiles@2
+    displayName: 'copy Redist dlls to CustomOperator Sample'
+    inputs:
+      SourceFolder: '$(System.DefaultWorkingDirectory)\buildArtifacts\$(ArtifactNameRedist)\bin'
+      Contents: |
+        windows.ai.machinelearning.dll
+        onnxruntime.dll
+        DirectML.dll
+        DirectML.pdb
+        onnxruntime.pdb
+        windows.ai.machinelearning.pdb
+      TargetFolder: '$(System.DefaultWorkingDirectory)\bin\$(BuildPlatform)\$(BuildConfiguration)\CustomOperator\'
+      OverWrite: true
+    condition: succeededOrFailed()
+
+  - task: PowerShell@2
+    displayName: 'Test CustomOperator Sample with Redist Dlls'
+    inputs:
+      targetType: 'inline'
+      script: |
+        $kitten = Resolve-Path ..\..\..\..\SharedContent\media\kitten_224.png
+        .\custom-operator-sample.exe relu
+        # registering new ops will only work for public sample on RS5 due to opset choice in code
+        $osinfo = Get-CimInstance Win32_OperatingSystem
+        $osinfo | Format-List *
+        $osversion = $osinfo.Version 
+        Write-Host "OS Version: $osversion"
+        if ($osversion -eq '10.0.17763') {
```

Comment:
We should really update the custom op sample to run correctly on 19H1 binaries. Or ideally be independent of binary (although I'm not sure this is possible). This may be out of scope of this PR but we should file some work for it.